web/src/containers/ApdStateProfileMedicaidOffice.js

```diff
@@ -21,68 +22,83 @@ class ApdStateProfile extends Component {
 
     return (
       <Fragment>
-        <h3 className="mt0">{t(`${dirTRoot}.title`)}</h3>
-        <div className="mb3">
-          <Input
+        <h4 className="ds-h4 ds-u-margin-bottom--0">
+          {t(`${dirTRoot}.title`)}
+        </h4>
+        <fieldset className="ds-u-margin--0 ds-u-padding--0 ds-u-border--0">
+          <legend className="sr-only">medicaid director details</legend>
+          <TextField
             name="apd-state-profile-mdname"
             label={t(`${dirTRoot}.labels.name`)}
             value={medicaidDirector.name}
             onChange={this.handleChange('medicaidDirector', 'name')}
           />
-          <Input
+          <TextField
             name="apd-state-profile-mdemail"
             label={t(`${dirTRoot}.labels.email`)}
             value={medicaidDirector.email}
             onChange={this.handleChange('medicaidDirector', 'email')}
           />
-          <Input
+          <TextField
             name="apd-state-profile-mdphone"
             label={t(`${dirTRoot}.labels.phone`)}
             value={medicaidDirector.phone}
             onChange={this.handleChange('medicaidDirector', 'phone')}
           />
-        </div>
-        <h3 className="mt0">{t(`${offTRoot}.title`)}</h3>
-        <div>
-          <Input
+        </fieldset>
+
+        <h4 className="ds-h4 ds-u-margin-bottom--0 ds-u-margin-top--5">
+          {t(`${offTRoot}.title`)}
+        </h4>
+        <fieldset className="ds-u-margin--0 ds-u-padding--0 ds-u-border--0">
+          <legend className="sr-only">medicaid office details</legend>
+          <TextField
             name="apd-state-profile-addr1"
             label={t(`${offTRoot}.labels.address1`)}
             value={medicaidOffice.address1}
             onChange={this.handleChange('medicaidOffice', 'address1')}
           />
-          <Input
+          <TextField
             name="apd-state-profile-addr2"
             label={t(`${offTRoot}.labels.address2`)}
+            hint="Optional"
             value={medicaidOffice.address2}
             onChange={this.handleChange('medicaidOffice', 'address2')}
           />
-          <div className="clearfix mxn1">
-            <div className="col col-12 sm-col-6 px1">
-              <Input
-                name="apd-state-profile-city"
-                label={t(`${offTRoot}.labels.city`)}
-                value={medicaidOffice.city}
-                onChange={this.handleChange('medicaidOffice', 'city')}
-              />
-            </div>
-            <div className="col col-6 sm-col-3 px1">
-              <Input
+          <div className="ds-l-row">
```

Comment:
This is the only way I could think of to get the two inputs on the same row.  One `TextField` is 9 columns, by default, which is where the 6 and 3 below come from.